telemetry/README.md

```diff
@@ -73,3 +74,4 @@ For specifics on how to consume the generators, see each IDE specific doc:
 
 -   [vscode](vscode/README.md)
 -   [jetbrains](jetbrains/README.md)
+-   [C#](csharp/README.md)
```

Comment:
Where is this used?

Reply:
The intent is to be open ended (not limited to just one application), but really it is used in the VS Toolkit

Reply:
oh my bad - I thought this was a link reference - i see it's just a list of links :(